### PR TITLE
cmake => 3.22.1, librhash => 1.4.2, libuv => 1.42.0, jsoncpp => 1.9.4

### PR DESCRIPTION
**Pull request by saltedcoffii**
Upgrade CMake and dependencies.

- [x] x86_64
- [x] i686
- [x] armv7l/aarch64

Build binaries in order: `librhash`, `libuv`, `jsoncpp`, `cmake`

### Run the following to get this pull request's changes locally for testing.
```
CREW_TESTING_REPO=https://github.com/saltedcoffii/chromebrew.git CREW_TESTING_BRANCH=cmake CREW_TESTING=1 crew update ; crew upgrade librhash libuv jsoncpp cmake
```

### CODE REVIEW

**Comment by satmandu** [comment]
Building cmake 3.21.4....

**Comment by satmandu** [comment]
The newly rebuilt version of cmake at 3.21.4 seems to work on armv7l...

**Comment by satmandu** [comment]
Let's do a rebuild of this as soon as the python3.10/libffi stuff is done and merged...

File: packages/cmake.rb [hunk 1]
```diff
@@ -3,26 +3,37 @@
 class Cmake < Package
   description 'CMake is an open-source, cross-platform family of tools designed to build, test and package software.'
   homepage 'https://cmake.org/'
-  @_ver = '3.20.5'
+  @_ver = '3.22.0'
```
Comment (uberhacker) [inline packages/cmake.rb:6]: Version 3.22.1 is now available.

**Comment by satmandu** [comment]
Yup. Will compile that next after these are done.

**Pull request merged**